pkg/util/cluster/cluster.go

```diff
@@ -484,6 +484,16 @@ func (c *Cluster) createCluster(ctx context.Context, vnetResourceGroup, clusterN
 	return c.openshiftclusters.CreateOrUpdateAndWait(ctx, vnetResourceGroup, clusterName, ocExt)
 }
 
+var insecureLocalClient *http.Client = &http.Client{
+	Transport: &http.Transport{
+		TLSClientConfig: &tls.Config{
+			InsecureSkipVerify: true,
+		},
+	},
+}
+
+var localDefaultURL string = "https://localhost:8443"
```

Comment:
What `var` and not `const`?

Reply:
Good point, `const` would indeed be the better choice here.